R/pkg/R/SQLContext.R

```diff
@@ -744,6 +746,9 @@ read.df.default <- function(path = NULL, source = NULL, schema = NULL, ...) {
   if (is.null(source)) {
     source <- getDefaultSqlSource()
   }
+  if (source == "csv" && is.null(options[["nullValue"]])) {
+    options[["nullValue"]] <- "NA"
```

Comment:
instead of hard coding this can we add a `na.strings` argument to `read.df` ? thats more similar to read.table in R https://stat.ethz.ch/R-manual/R-devel/library/utils/html/read.table.html